packages/react-components/src/components/PictureInPictureInPicture/PictureInPictureInPicture.tsx

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { VideoTileProps } from '../VideoTile';
+
+/**
+ * Props for {@link _PictureInPictureInPicture} component.
+ *
+ * @internal
+ */
+export interface _PictureInPictureInPictureProps {
```

Comment:
I'm assuming this props definition is also only for demonstration purposes.

Reply:
Yeah, it'd v. much subject to change depending on implementation - I'll simplify for this PR

Reply:
updated